public/home/ClientIcons.tsx

```diff
@@ -4,30 +4,33 @@ import Link from 'next/link';
 
 export default function ClientIcons() {
   return (
-    <div className="w-full h-full">
-      <div className="flex flex-row items-center mb-14  justify-between">
-        <Link href={'https://www.22seven.com'}>
+    <div className="w-full opac h-full">
+      <div className="flex flex-row items-center mb-14  justify-between ">
+        <Link
+          className="bg-[#f1f1f1] opacity-50 hover:opacity-100 "
+          href={'https://www.22seven.com'}
+        >
           <div>
             <picture>
               <img src="/client-icons/22seven.svg" alt="22seven" />
             </picture>
           </div>
         </Link>
-        <Link href={'https://bank.tech'}>
+        <Link className="bg-[#f1f1f1] opacity-50 hover:opacity-100 " href={'https://bank.tech'}>
```

Comment:
no need for inline tailwind color, there is a color name alias for this in the config. I think its called grey